projects/components/src/stepper/stepper.component.scss

```diff
@@ -0,0 +1,56 @@
+@import 'mixins';
+
+:host {
+  ::ng-deep {
+    .mat-step-header {
+      .mat-step-label {
+        @include overline($gray-5);
+        text-transform: uppercase;
+      }
+
+      .mat-step-label-selected {
+        @include overline($gray-7);
+        text-transform: uppercase;
+      }
+
+      .mat-step-icon {
+        @include overline(white);
+        @include frame-border;
+        border-color: $blue-4;
+        color: $blue-4;
+      }
+
+      .mat-step-icon-selected {
+        background-color: $blue-4;
+        color: white;
+      }
+    }
+
+    .mat-vertical-stepper-header {
+      padding: 24px;
+    }
+
+    .mat-vertical-content-container.mat-stepper-vertical-line::before {
+      top: -16px;
+      bottom: -16px;
+      border-left-color: $gray-9;
```

Comment:
Does it fine just set the border-left-color here?

Reply:
Removed that block